auacm/app/modules/problem_manager/views.py

```diff
@@ -255,7 +258,7 @@ def update_problem(identifier):    # pylint: disable=too-many-branches
     data = database.session.query(ProblemData).filter(ProblemData.pid == pid).first()
     if 'name' in request.form:
         problem.name = request.form['name'][:32]
-        problem.shortname = request.form['name'][:32].replace(' ', '').lower()
+        problem.shortname = request.form['name'][:32].replace(' ', '').replace('(', '').replace(')', '').lower()
```

Comment:
+1, I actually think that we shouldn't be making any backend changes in this PR; it's big enough as-is.